ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos-device-documentation.j2

```diff
@@ -213,6 +214,10 @@
 
 {% include 'documentation/vxlan-interface.j2' %}
 
+## ISIS Interfaces
+
+{% include 'documentation/isis-interfaces.j2' %}
```

Comment:
Not sure if it would not be better to put ISIS table in their respective Interface section (Ethernet / vlan / loopback) to make all information about interfaces in the same place.

Reply:
I think you right

Reply:
done. they are in both interfaces doc and router isis doc